### DISCUSSION TITLE
Recovery Phrase     Invalid Mnemonic ChecksumMismatch

### DISCUSSION THREAD

**Discussion by OKRupp** [iotaledger/firefly#6341] [Help & Support]
I have downloaded the new Wallet and wanted to restore the old wallet with the 24 word phrase
After entering the 24 words I receive the error message
 " Invalid checksum "

What do I need to do / change pls ? 
Thanks

**Comment by MiKeF05** [comment]
You may have an typo in one of the words.

**Comment by OKRupp** [comment]
I had a typo in the beginning (error message) and corrected it ...Then the
error code charged to Checksum Mismatch

MiKeF05 ***@***.***> schrieb am Sa., 1. Apr. 2023, 15:33:

**Reply by JannemanDev** [reply]
Did you type or copy pasted the words?

**Comment by OKRupp** [comment]
I Typed


JannemanDev ***@***.***> schrieb am Sa., 1. Apr. 2023, 17:55:

**Comment by OKRupp** [comment]
thanks all...
I deletetd the wallet...reinstalled..entered the 24 words without a typo...now works fine
seems the installation process dont like any typos (happened during the first time, but even after correction the error stayed)
